UKHomeOffice/nmsw-ui
# NMSW-633 Update error mapping

# Ticket



----

## AC

the default mandatory field text coming from API response is now 'field required' so error mapping for FAL1 needs updating

----

## To test

Upload a FAL1 with empty fields
You should see error messages that reference the field name


----

## Notes
